internal/pkg/cli/deploy/env.go

```diff
@@ -413,6 +405,31 @@ func (d *envDeployer) buildStackInput(in *DeployEnvironmentInput) (*cfnstack.Env
 	}, nil
 }
 
+func (d *envDeployer) buildAddonsInput(region, bucket, fallbackURL string) (*cfnstack.Addons, error) {
```

Comment:
oo I see. The `fallbackURL` is not empty if we've uploaded the assets. 

Can we rename it to something like `uploadedURL`? `fallbackURL` sounds like it should be the last URL to use if nothing else works.